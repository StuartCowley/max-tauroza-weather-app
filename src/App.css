@import url('https://fonts.googleapis.com/css2?family=Goldman&display=swap');

.title {
  font-family: 'Goldman';
}
