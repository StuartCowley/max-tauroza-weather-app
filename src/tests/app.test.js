import React from "react";
import { render /*screen, waitFor*/ } from "@testing-library/react";
// import axios from "axios";
import App from "../../components/app";

describe("App", () => {
  it("renders app components correctly", async () => {
    // const mockedData = {
    //   location: {
    //     city: "Test city",
    //     country: "Test country",
    //   },
    //   forecasts: [
    //     {
    //       date: 111111,
    //       temperature: { max: 2, min: 1 },
    //       wind: { speed: 1, direction: "test direction" },
    //       humidity: 1,
    //       description: "Test description",
    //       icon: 1,
    //     },
    //   ],
    // };

    // jest.spyOn(axios, "get").mockResolvedValue({ mockedData });
    // const { asFragment } = render(<App />);

    // expect(
    //   await screen.findByText(/Test city, Test country/i)
    // ).toBeInTheDocument();
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});
