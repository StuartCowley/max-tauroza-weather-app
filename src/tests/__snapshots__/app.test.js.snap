// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders app components correctly 1`] = `
<DocumentFragment>
  <div
    class="weather-app"
  >
    <h1>
      , 
    </h1>
    <div
      class="search-form"
    >
      <label
        for="location-search"
      >
        Change location
        <input
          id="location-search"
          placeholder="Enter a town or city"
          type="text"
          value=""
        />
        <button
          type="submit"
        >
          Search
        </button>
      </label>
    </div>
    <div
      class="forecast-summary-list"
    />
  </div>
</DocumentFragment>
`;
